task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs"
   name: fbc-validation
 spec:
+  description: >-
+    Validate the fbc image
```

Comment:
What is going on in `source /utils.sh`? 

Should this say "Validate the file-based-catalog (fbc) image with opm"?

Reply:
After you get a response, @ralphbean, I'll suggest edits.

Reply:
We have functions in script https://github.com/redhat-appstudio/hacbs-test/blob/main/test/utils.sh which is built into `quay.io/redhat-appstudio/hacbs-test`, so we need `source /utils.sh` before using these functions

Reply:
Updated to `Validates the file-based-catalog (FBC) image with opm.`,  thanks.